app/Http/Controllers/LikeController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Post;
+use App\Models\Like;
+
+use Auth;
+//use Illuminate\Support\Facades\Auth as FacadesAuth;
+
+class LikeController extends Controller
+{
+    public function store(Post $post)
+{
+    $user = auth()->user();
+    
+    if (!$user->hasLiked($post)) {
+        $like = new Like();
+        $like->user_id = $user->id;
+        $post->likes()->save($like);
```

Comment:
```suggestion
        $like->post_id = $post->id;
        $like->save();
```
としたほうが、$likeを登録していることがわかりやすいです。